src/ripper/ripcddialog.cpp

```diff
@@ -243,6 +254,7 @@ void RipCDDialog::ClickedRipButton() {
 
   SetWorking(true);
   ripper->Start();
+  transcoding_progress_timer_.start(kTranscodingProgressIntervalMs);
```

Comment:
Technically, there's a race condition here, though you should never hit it with the 500ms timer. If no tracks were added, the timer callback would call Ripper::GetProgress and divide by zero.

Reply:
Good catch! I added a check in `Ripper::GetProgress` to prevent division by zero now.